### ISSUE TITLE
Question: Befaco 7U Case MIDI DIN ports

### ISSUE THREAD

**Issue by SacredSkull** [Befaco/MIDIThing2#17]
Hey,

Considering getting this module, and there doesn't seem to be a way to connect it directly to the large DIN ports on the Befaco 7U case - any particular reason? I would be getting the DIY version so could quite easily just wire the 3.5mm jack pins to the header on the case, but just wondering why this wasn't added in the first place?

I believe the Intellijel modules are the only modules actually using these header pins - or... does the internal USB header on the Befaco case map to the MIDI DIN ports?

In other words, would plugging in the MIDI Thing V2 into the internal USB port on the Befaco 7U case allow the DIN ports to work - if so I have answered my own question!

**Comment by xustafu** [comment]
As MIDI Thing is a 3U module we did not considerd this option, as the copnnectors are accesible i nthe front. BUT, you can connect to the back of the case using a micro-USB cable and use the USB back port as a MIDI device.

We are finishing up a 1U version of the module that will use full use of the back ports of the case. Stay tunned!
